docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -26,6 +26,7 @@ metrics based on the tracing data.
   * [Grizzly](https://github.com/eclipse-ee4j/grizzly) 2.3.x -> 4.x
   * [WebSphere Liberty](https://github.com/OpenLiberty/open-liberty) 23.x
   * [Netty HTTP](https://github.com/netty/netty) 4.1.x (Optional²)
+  * [Solon](https://github.com/opensolon/solon) 2.7.x -> 2.8.x
```

Comment:
```suggestion
  * [Solon](https://github.com/opensolon/solon) 2.7.x -> 2.8.x
```

Only 2.7 and 2.8 supported? I can see 2.6 was released 5 months ago, and 2.5 was 10 months ago.

I don't know solon community style.
Are the latest two enough?

Reply:
solon is in rapid development, so updates are iterated rather quickly. I will explain the support for historical versions in the solon community.